app/src/main/java/com/yeonae/chamelezone/view/place/PlaceDetailActivity.kt

```diff
@@ -114,7 +110,7 @@ class PlaceDetailActivity : AppCompatActivity(),
         )
         presenter.checkLogin()
 
-        btn_back.setColorFilter(Color.parseColor("#F5DA81"), PorterDuff.Mode.SRC_IN)
+        btn_back.setColorFilter(Color.parseColor("#F0843F"), PorterDuff.Mode.SRC_IN)
```

Comment:
```suggestion
        btn_back.setColorFilter(getColor(R.color.colorBlack), PorterDuff.Mode.SRC_IN)
```
res에 저장된 COLOR를 사용하는 것이 좋을 것 같아요.

Reply:
수정했습니다
bf90490